src/brogue/Globals.c

```diff
@@ -22,113 +22,50 @@
  */
```

Comment:
This contains constants which might want to be set by variants but aren't used in RapidBrogue. Variables that are set in RapidBrogue and Brogue differently have been migrated from Globals.c to GlobalsBrogue.c. If desired, I could complete the job and move all remaining variables from Globals.c to GlobalsBrogue.c. In this case, GlobalsRapidBrogue.c would reference many GlobalsBrogue.c tables where the behaviour is desired to be the same (e.g. monsterCatalog = monsterCatalog_Brogue);

Reply:
I think I'd want some general variant API changes before that. Having a core/variant separation is fine at this point.

>Variables that are set in RapidBrogue and Brogue differently have been migrated from Globals.c to GlobalsBrogue.c.

What do you mean by this? In this file are the global tables that variants override. Or do you mean that the actual data has been moved to the per-variant globals files?

Reply:
My statement is not very clear but I don't understand your question :)
What I meant was:

Where a table (array of structs declared globally) in Globals.c has different data in Brogue and RapidBrogue, new tables containing the variant-specific data have been created in GlobalsBrogue.c and GlobalsRapidBrogue.c. The table containing data in Globals.c has been replaced by a pointer to the table within Globals.c (again declared globally). On initialization of a variant, this global pointer is set to the variant specific table. 

As an aside, these global pointers are declared as `const` where possible - when the table contains information used in the creation of game objects only.

Reply:
Nice, that makes sense to me 👍